s2i/bin/run

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash -e
-source /opt/app-root/etc/dotnet_environment
 
-echo "---> Running application ..."
-exec "$DOTNET_RUN_SCRIPT"
+echo "---> Starting the application ..."
+
+if [ -f "./gradlew" ]; then
```

Comment:
This check would be better off included into `gradle_environment` as it's used in both the assemble and run script